### DISCUSSION TITLE
How can I make a current for the App Router?

### DISCUSSION THREAD

**Discussion by Nurseit05** [i18next/next-i18next#2268] [Q&A]
I have nextjs 14 v on App router, in the example for pages router. I did according to the documentation for app router but it didn't work and it requires to use on client side, I want to use on server side.

**Comment by adrai** [comment]
Since v16 (March 2026) next-i18next ships first-class App Router support: `getT()` for Server Components, `useT()` for Client Components, `createProxy()` for locale detection and routing. Docs: https://github.com/i18next/next-i18next#app-router-setup · Background: https://www.locize.com/blog/next-i18next-v16?utm_source=github&utm_medium=discussion_2268
